src/server.js

```diff
@@ -121,7 +110,7 @@ function index(req, res, next) {
 			pkg,
```

Comment:
So, this merge of `data` and `pkg` is why `client/index.html` L337 and L338 can make use of `version`. In other words, version being displayed in the UI will come from the `package.json` file and not your `getVersion()` function.
Is that expected?
Do you think it could create some confusion if `getVersion()` and the `version` field of the `package.json` file do not return the same value (which they probably don't in most cases)?